src/app/payment/services/checkout.service.ts

```diff
@@ -67,14 +67,15 @@ const getClientSecretForSubscriptionIntent = async (
   priceId: string,
   token: string,
   currency: string,
+  quantity = 1,
   promoCodeId?: string,
 ): Promise<ClientSecretData & { subscriptionId?: string; paymentIntentId?: string }> => {
   const {
     type: paymentType,
     clientSecret: client_secret,
     subscriptionId,
     paymentIntentId,
-  } = await paymentService.createSubscription(customerId, priceId, token, currency, promoCodeId);
+  } = await paymentService.createSubscription(customerId, priceId, token, currency, quantity, promoCodeId);
```

Comment:
what does the `quantity` param refer to?

Reply:
Maybe you should use a better name for it. This parameter indicates the number of subscriptions the user wants to purchase for his business plan.